domains/hanzas.thedev.me.json

```diff
@@ -0,0 +1,37 @@
+{
+    "subdomain": "hanzas",
+    "domain": "thedev.me",
+    "email_or_discord": "josephandrew1@logicstreak.com",
+    "github_username": "hanzas09",
+    "description": "website of hanzas web development service",
+    "records": {
+        "A": [
+            "185.199.108.153",
+            "185.199.109.153",
+            "185.199.110.153",
+            "185.199.111.153"
+        ],
+        "AAAA": [
+            "2606:50c0:8000::153",
+            "2606:50c0:8001::153",
+            "2606:50c0:8002::153",
+            "2606:50c0:8003::153"
+        ],
+        "CNAME": [
+            "hanzas09.github.io"
+        ],
+        "NS": [
+            "ns1.example.com",
+            "ns2.example.com"
+        ],
```

Comment:
Remove the trailing dot in your MX records and all should be good to merge.

Reply:
Ok Done.